clang/test/CodeGenSYCL/reqd-sub-group-size-host.cpp

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-host -triple -x86_64-unknown-linux-gnu -disable-llvm-passes %s -emit-llvm -o -  | FileCheck %s
+
+class Functor16 {
```

Comment:
Nit: Can you rename the functors to be consistent. I initially thought Functor16 was named with '16' to indicate the size argument of reqd_sub_group_size. But the other classes below are not named this way.

Reply:
Fixed that

Reply:
Thanks!